exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -237,71 +281,81 @@
-    new ExecutionControls.ControlsOptionValidator(DRILLBIT_CONTROL_INJECTIONS, ExecutionControls.DEFAULT_CONTROLS, 1);
+  public static final String DRILLBIT_CONTROL_INJECTIONS = "drill.exec.testing.controls";
+  public static final OptionValidator DRILLBIT_CONTROLS_VALIDATOR =
+      new ExecutionControls.ControlsOptionValidator(DRILLBIT_CONTROL_INJECTIONS, ExecutionControls.DEFAULT_CONTROLS, 1);
 
-  String NEW_VIEW_DEFAULT_PERMS_KEY = "new_view_default_permissions";
-  OptionValidator NEW_VIEW_DEFAULT_PERMS_VALIDATOR =
-      new StringValidator(NEW_VIEW_DEFAULT_PERMS_KEY, "700");
+  public static final String NEW_VIEW_DEFAULT_PERMS_KEY = "new_view_default_permissions";
+  public static final OptionValidator NEW_VIEW_DEFAULT_PERMS_VALIDATOR = new StringValidator(
+      NEW_VIEW_DEFAULT_PERMS_KEY, "700", "Sets view permissions using an octal code in the Unix tradition.");
 
-  String CTAS_PARTITIONING_HASH_DISTRIBUTE = "store.partition.hash_distribute";
-  BooleanValidator CTAS_PARTITIONING_HASH_DISTRIBUTE_VALIDATOR = new BooleanValidator(CTAS_PARTITIONING_HASH_DISTRIBUTE, false);
+  public static final String CTAS_PARTITIONING_HASH_DISTRIBUTE = "store.partition.hash_distribute";
+  public static final BooleanValidator CTAS_PARTITIONING_HASH_DISTRIBUTE_VALIDATOR = new BooleanValidator(
+      CTAS_PARTITIONING_HASH_DISTRIBUTE, false);
 
-  String ENABLE_BULK_LOAD_TABLE_LIST_KEY = "exec.enable_bulk_load_table_list";
-  BooleanValidator ENABLE_BULK_LOAD_TABLE_LIST = new BooleanValidator(ENABLE_BULK_LOAD_TABLE_LIST_KEY, false);
+  public static final String ENABLE_BULK_LOAD_TABLE_LIST_KEY = "exec.enable_bulk_load_table_list";
+  public static final BooleanValidator ENABLE_BULK_LOAD_TABLE_LIST = new BooleanValidator(
+      ENABLE_BULK_LOAD_TABLE_LIST_KEY, false);
 
   /**
    * Option whose value is a comma separated list of admin usernames. Admin users are users who have special privileges
    * such as changing system options.
    */
-  String ADMIN_USERS_KEY = "security.admin.users";
-  StringValidator ADMIN_USERS_VALIDATOR =
+  public static final String ADMIN_USERS_KEY = "security.admin.users";
+  public static final StringValidator ADMIN_USERS_VALIDATOR =
       new AdminOptionValidator(ADMIN_USERS_KEY, ImpersonationUtil.getProcessUserName());
 
   /**
    * Option whose value is a comma separated list of admin usergroups.
    */
-  String ADMIN_USER_GROUPS_KEY = "security.admin.user_groups";
-  StringValidator ADMIN_USER_GROUPS_VALIDATOR = new AdminOptionValidator(ADMIN_USER_GROUPS_KEY, "");
+  public static final String ADMIN_USER_GROUPS_KEY = "security.admin.user_groups";
+  public static final StringValidator ADMIN_USER_GROUPS_VALIDATOR = new AdminOptionValidator(ADMIN_USER_GROUPS_KEY, "");
 
   /**
-   * Option whose value is a string representing list of inbound impersonation policies.
+   * Option whose value is a public static final String representing list of inbound impersonation policies.
```

Comment:
No need to state the type in the Javadoc comment; the type declaration will appear in the generated docs. So, must say "List of inbound..."

Then, below, in the example format, the text is interpreted as HTML. So, add the required directives:

    * <pre>[
    *    ....
    * [</pre>